readme.txt

```diff
@@ -30,8 +30,54 @@ Extract the contents of the ZIP and upload the contents to the `/wp-content/plug
 
 Data is coming from [The Green Web Foundation](https://www.thegreenwebfoundation.org/) API.
 
+== Security ==
+
+For security or privacy concerns, please contact us at [wordpress @ wpsysadmin . com].
```

Comment:
In the Plugin Directory, the content from readme.txt files is processed as Markdown, but it doesn't automatically convert email addresses into clickable mailto links unless we format them that way. Additionally, broken-up email addresses like [wordpress @ wpsysadmin . com] may confuse users and are not clickable.

Hence I would recommend changing this to:
```suggestion
For security or privacy concerns, please contact us at wordpress@wpsysadmin.com.
```
or
```suggestion
For security or privacy concerns, please contact us at [wordpress@wpsysadmin.com](mailto:wordpress@wpsysadmin.com).
```

I would also like to add that email addresses not being formatted as such badly affects the accessibility for disabled users.